### PR TITLE
Documentation on setting node name with Kubeadm

### PR DESCRIPTION
**Pull request by liztio**
Updates to kubeadm init docs about `--node-name` changes.

Requested by @luxas on kubernetes/kubernetes#64706

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 1]
```diff
@@ -348,6 +348,14 @@ In order to set up a cluster where the master and worker nodes communicate with
 
 3. Finally, when you run `kubeadm join`, make sure you provide the private IP of the API server addressed as defined in step 1.
 
+### Setting the node name name
```
Comment (DStorck) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: did you mean to write name twice ?

Comment (liztio) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: whoops. Overzealous find/replace

**Comment by mistyhacks** [comment]
ping @Bradamant3

**Comment by Bradamant3** [comment]
/hold 

this was filed in error against the master branch; per the k/k PR, it's not in until 1.11. At this point, I'm inclined to wait until after the release to merge it.

**Comment by mistyhacks** [comment]
/hold cancel

🎉

**Pull request merged**